package.json

```diff
@@ -20,17 +20,7 @@
     "url": "https://github.com/green-fox-academy/JSA-Opal-ShenZhen/issues"
   },
   "homepage": "https://github.com/green-fox-academy/JSA-Opal-ShenZhen#readme",
-  "devDependencies": {
-    "eslint": "^6.6.0",
-    "eslint-config-airbnb": "^18.0.1",
-    "eslint-config-prettier": "^6.5.0",
-    "eslint-plugin-import": "^2.18.2",
-    "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-prettier": "^3.1.1",
-    "eslint-plugin-react": "^7.16.0",
-    "eslint-plugin-react-hooks": "^1.7.0",
-    "eslint-plugin-react-native": "^3.8.1",
-    "pre-commit": "^1.2.2",
-    "prettier": "^1.19.1"
+  "dependencies": {
+    "pre-commit": "^1.2.2"
```

Comment:
@hondahmd you dont use this dependency anywhere

Reply:
@adamgyulavari well it's used for lint before commit, and used as [npm script](https://github.com/green-fox-academy/JSA-Opal-ShenZhen/blob/honda-issue8-lint/package.json#L10)